assets/stylesheets/bookingsync/_sweetalert.scss

```diff
@@ -0,0 +1,7 @@
+.swal2-popup {
+  font-family: $font-family-sans-serif;
+}
+
+.swal2-icon {
+  display: none !important;
+}
```

Comment:
We don't need to hide the icon in css. We just don't pass the `type` when we use the alert and the icon won't be shown.